src/components/SignMessage.vue

```diff
@@ -210,20 +211,28 @@ function isDeprecatedMethod() {
     </div>
     <div v-else class="flex flex-col gap-1">
       <div class="text-sm">Message</div>
-      <SignMessageAdvancedInfo
-        :info="advancedInfo(request.request.method, request.request.params)"
-      />
-      <!-- <div
-        v-if="method !== 'eth_signTransaction'"
-        class="flex justify-center mt-4"
-      >
+      <div v-if="request.request.method === 'mvx_signTransaction'">
         <div
-          class="flex bg-white-100 border-1 border border-gray-300 dark:bg-gray-300 rounded-sm p-2 text-xs gap-1 dark:text-gray-100"
+          v-for="[key, value] in Object.entries(
+            request.request.params.transaction
+          )"
+          :key="key"
+          class="flex justify-between gap-4"
         >
-          <img src="@/assets/images/info-circle.svg" />
-          <span>You're not going to be charged</span>
+          <span class="w-[120px] capitalize">{{ key }}</span>
+          <span>
```

Comment:
add value in `title` attribute so it shows on hover

Reply:
[done](https://github.com/arcana-network/wallet-ui/pull/614/commits/20003abac4bb30cf0a7ce52c09b9dca2e0a095a1)